packages/plugin-board/test/integration/spec/board.js

```diff
@@ -0,0 +1,414 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import '../..';
+
+import {assert} from '@ciscospark/test-helper-chai';
+import CiscoSpark from '@ciscospark/spark-core';
+import testUsers from '@ciscospark/test-helper-test-users';
+import {flaky} from '@ciscospark/test-helper-mocha';
+import fh from '@ciscospark/test-helper-file';
+import {find, map} from 'lodash';
+import uuid from 'uuid';
+
+function generateTonsOfContents(numOfContents) {
+  const contents = [];
+
+  for (let i = 0; i < numOfContents; i++) {
+    contents.push({
+      type: `curve`,
+      payload: JSON.stringify({id: i, type: `curve`})
+    });
+  }
+  return contents;
+}
+
+function boardChannelToMercuryBinding(channelId) {
+  // make channelId mercury compatible replace `-` with `.` and `_` with `#`
+  return channelId.replace(/-/g, `.`).replace(/_/g, `#`);
+}
+
+describe(`plugin-board`, function() {
+  this.timeout(30000);
+
+  let board, conversation, fixture, participants;
+  const mercuryBindingsPrefix = `board.`;
+
+  // create users
```

Comment:
Interesting. In my experience, as long as blocks return promises, they execute in series. 

when I wrote the conversation tests a while ago, i added the ensureConversation function rather than simply a before block to try to speed up the test suite while not breaking the ability to run just a single test for debugging. it didn't work as well as i'd hoped, so i have no objection to removing that construct.